samples/notifications.js

```diff
@@ -0,0 +1,198 @@
+/**
+ * Copyright 2017, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * This application demonstrates how to perform basic operations on files with
+ * the Google Cloud Storage API.
+ *
+ * For more information, see the README.md under /storage and the documentation
+ * at https://cloud.google.com/storage/docs.
+ */
+
+'use strict';
+
+function createNotification(bucketName, topic) {
+  // [START storage_create_notification]
+  // Imports the Google Cloud client library
+  const Storage = require('@google-cloud/storage');
+
+  // The name of the bucket to access, e.g. "my-bucket"
+  // const bucketName = "my-bucket";
+
+  // The name of the topic to which this subscription publishes.
+  // const topic = "my-topic";
+
+  // Instantiates a client
+  const storage = Storage();
+
+  // Lists files in the bucket
+  storage
+    .bucket(bucketName)
+    .createNotification(topic)
+    .then(() => {
+      console.log('Notification subscription created.');
+    })
+    .catch(err => {
+      console.error('ERROR:', err);
+    });
+  // [END storage_create_notification]
+}
+
+function listNotifications(bucketName) {
+  // [START storage_list_notifications]
+  // Imports the Google Cloud client library
+  const Storage = require('@google-cloud/storage');
+
+  // The name of the bucket to access, e.g. "my-bucket"
+  // const bucketName = "my-bucket";
+
+  // Instantiates a client
+  const storage = Storage();
+
+  // Lists files in the bucket
+  storage
+    .bucket(bucketName)
+    .getNotifications()
+    .then(results => {
+      const notifications = results[0];
+
+      console.log('Notifications:');
+      notifications.forEach(notification => {
+        console.log(notification.id);
+      });
+    })
+    .catch(err => {
+      console.log('ERROR:', err);
+    });
+  // [END storage_list_notifications]
+}
+
+function getMetadata(bucketName, notificationId) {
+  // [START storage_delete_notification]
```

Comment:
I think this tag should be `[START storage_get_metadata]` to match the closing tag, although a tag like `[START storage_notifications_get_metadata]` (or something mentioning the `notifications` part) would help separate it from the other examples in the other parts of the API.

Reply:
Whoops, nice catch!